add-on/chrome/bootstrap.js

```diff
@@ -1431,6 +1473,11 @@ function shutdown(data, reason) {
   // Stop waiting for browser windows to open.
   wm.removeListener(WindowListener);
 
+  // unregister about: handlers
+  AboutLoop.conversation.unregister(); // eslint-disable-line no-undef
+  AboutLoop.panel.unregister(); // eslint-disable-line no-undef
+  AboutLoop.toc.unregister(); // eslint-disable-line no-undef
```

Comment:
I can't see the comment you're talking about

https://github.com/mozilla/loop/pull/329